src/components/molecules/Tag/Tag.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import PlusIcon from '../../atoms/PlusIcon';
+import CloseIcon from '../../atoms/CloseIcon';
+import theme from '../../../style/theme';
+
+const Wrapper = styled.div`
+  display: inline-flex;
+  border-radius: 2vh;
+  padding: 0.9vh 1.8vh;
+  background-color: ${({ isAddingTag, tagColor }) =>
+    isAddingTag ? `white` : tagColor};
+  border: ${({ isAddingTag, tagColor }) =>
+    isAddingTag ? `solid 0.2vh ${tagColor}` : ''};
+`;
+const Content = styled.p`
+  color: ${({ contentColor }) => contentColor};
+  display: inline-block;
+  font-size: 2vh;
+  margin-top: 0.3vh;
+  margin-left: ${({ isAddingTag }) => isAddingTag && `1.4vh`};
+  margin-right: ${({ isAddingTag }) => !isAddingTag && `1.4vh`};
+`;
+
+export default function Tag({
+  func,
+  tagColor,
+  contentColor,
+  isAddingTag,
+  closeTagText,
+}) {
+  if (isAddingTag) {
+    return (
+      <Wrapper onClick={func} tagColor={tagColor} isAddingTag={isAddingTag}>
+        <PlusIcon color={contentColor} />
+        <Content contentColor={contentColor} isAddingTag={isAddingTag}>
+          추가
+        </Content>
+      </Wrapper>
+    );
+  }
+
+  return (
+    <Wrapper onClick={func} tagColor={tagColor} isAddingTag={isAddingTag}>
+      <Content contentColor={contentColor} isAddingTag={isAddingTag}>
+        {closeTagText}
+      </Content>
+      <CloseIcon color={contentColor} />
+    </Wrapper>
+  );
```

Comment:
```suggestion
  return (
    <Wrapper onClick={func} tagColor={tagColor} isAddingTag={isAddingTag}>
      {isAddingTag && <PlusIcon color={contentColor} />}
      <Content contentColor={contentColor} isAddingTag={isAddingTag}>
        {!isAddingTag ? '추가' : closeTagText}
      </Content>
      {!isAddingTag && <CloseIcon color={contentColor} />}
    </Wrapper>
  );
```
이렇게 축약할 수 있을 것 같습니다!

Reply:
확인했습니다! 리뷰 감사드립니다🎉